catalog/java/io/material/catalog/datepicker/DatePickerMainDemoFragment.java

```diff
@@ -163,13 +166,30 @@ public View onCreateDemoView(
             builder.setCalendarConstraints(constraintsBuilder.build());
             MaterialDatePicker<?> picker = builder.build();
             addSnackBarListeners(picker);
-            picker.show(getChildFragmentManager(), picker.toString());
+            picker.show(getChildFragmentManager(), "MaterialDatePicker");
```

Comment:
Why did you change picker.toString() to a string constant?

Reply:
I changed it to find the fragment by findFragmentByTag(***"MaterialDatePicker"***).